### PR TITLE
Fix tag and manufacturer selection

### LINKED ISSUE #96: Can't create a new tag when creating/editing a product if a similar tag exists
Right now its not possible to create a new tag, if a similar named tag already exists. 

E.g.:

The tag "alcohol-free" already exists and I want to create the tag "alcohol" it's not possible, because the modal suggests to use "alcohol-free". There is no way to create this new tag.

### PR DESCRIPTION
**Pull request by theErikss0n**
fixes #96 #40

obsoletes #165

### CODE REVIEW

File: app/src/components/admin-page/edit-product-modal.vue [hunk 1]
```diff
@@ -69,22 +69,14 @@
                                 <v-col>
                                     <v-autocomplete
                                         v-model="editProduct.manufacturerID"
-                                        :items="manufacturers"
+                                        :items="manufacturerSearchItems"
```
Comment (andresterba) [inline app/src/components/admin-page/edit-product-modal.vue:72]: Everything works just perfekt for tags!
Well done :tada: 

But somethings seems to be wrong for manufacturers as I can't create a single one :shrug:

**Review by andresterba — CHANGES_REQUESTED** [review]

**Comment by theErikss0n** [comment]
Manufacturers are working now :tada:

File: app/src/components/admin-page/edit-product-modal.vue (hunk 1 above)
Comment (theErikss0n) [reply app/src/components/admin-page/edit-product-modal.vue:72]: Should be fixed by new commit

**Review by andresterba — APPROVED** [review]
Good job getting this to work! :tada:

**Pull request merged**